python/ray/serve/_private/constants.py

```diff
@@ -29,7 +29,7 @@
 DEFAULT_GRPC_PORT = 9000
 
 #: Default Serve application name
-SERVE_DEFAULT_APP_NAME = ""
+SERVE_DEFAULT_APP_NAME = "DEFAULT_RAY_SERVE_APP"
```

Comment:
this is potentially user facing so this name is a bit ugly, should we just make it "default" ?